### DISCUSSION TITLE
Error of lammps + GPU using flare potential

### DISCUSSION THREAD

**Discussion by shihao-code** [mir-group/flare#339] [Q&A] [answered]
**As I used flare (v_1.3.3) potential for lammps (22 Dec 2022) + GPU calculation (Kokkos) (i.e., lmp_mpi -k on g 1 -sf kk -package kokkos newton on neigh full -in *.in > olog.txt), an error was obtained, and "nan" was output.**

Step         PotEng         Volume         Cella          Cellb          Cellc        CellAlpha       CellBeta      CellGamma        Press           Pxx            Pyy            Pzz            Pxy            Pxz            Pyz          c_eatoms
         0  -nan            268717.77      64.530564      64.530564      64.530564      90             90             90            -nan           -nan           -nan           -nan           -nan           -nan           -nan           -nan

**But, if I don't used GPU with the some lammps (i.e., lmp_mpi -in *.in > olog.txt), there is not error.**

Step         PotEng         Volume         Cella          Cellb          Cellc        CellAlpha       CellBeta      CellGamma        Press           Pxx            Pyy            Pzz            Pxy            Pxz            Pyz          c_eatoms
         0  -1707.7557      268717.77      64.530564      64.530564      64.530564      90             90             90            -9344.8769     -9344.8769     -9344.8769     -9344.8769     -2.0916633e-10  3.4853549e-10  2.1048608e-10 -1707.7557

**Please see the attached log files of lammps for detail.**

[log.lammps_CPU.txt](https://github.com/mir-group/flare/files/10368864/log.lammps_CPU.txt)
[log.lammps_GPU.txt](https://github.com/mir-group/flare/files/10368865/log.lammps_GPU.txt)

Thank you sincerely！

**Comment by YuuuXie** [comment, marked as answer]
Hi @shihao-code , I think your problem is related to the issue here: https://github.com/mir-group/flare_pp/issues/43 (@anjohan)

(A side note, the issue is opened in the flare_pp repo, and we have merged flare_pp into flare. The current repo (flare) is the one that we are actively maintaining.)
